articles/article-058-mutex.md

```diff
@@ -0,0 +1,25 @@
+<!-- BEGIN TITLE -->
+# Semaphores prevent crashes
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Imagine a railway intersection where only one train can pass at a time. If any other train tries to pass the intersection while it's already being used, a crash happens.
```

Comment:
Language: "If another train tries to pass" instead of "if any other train"?
Language: "the trains will crash" instead of "a crash happens"?